src/main/scala/org/ergoplatform/nodeView/wallet/persistence/WalletRegistry.scala

```diff
@@ -31,6 +31,10 @@ class WalletRegistry(store: HybridLDBKVStore)(ws: WalletSettings) extends Scorex
 
   private val keepHistory = ws.keepSpentBoxes
 
+  def close(): Unit = {
```

Comment:
If this is overridden method, please mark is `override` explicitly. Otherwise provide ScalaDoc for the public method.

Reply:
It is not overridden method,  ScalaDoc added